.github/workflows/tests.yml

```diff
@@ -159,32 +163,45 @@ jobs:
       with:
         distribution: 'temurin'
         java-version: ${{ matrix.java-version }}
-    - name: Set up Gradle 7.3 # must be done manually due to no gradle build in capa
+    - name: Set up Gradle ${{ matrix.gradle-version }} # must be done manually due to no gradle build in capa
       run: |
         mkdir /opt/gradle
-        wget "https://services.gradle.org/distributions/gradle-7.3-bin.zip" -O /opt/gradle/gradle-7.3.zip
-        unzip /opt/gradle/gradle-7.3.zip -d /opt/gradle
-    - name: Install Ghidra 10.3
+        wget "https://services.gradle.org/distributions/gradle-${{ matrix.gradle-version }}-bin.zip" -O /opt/gradle/gradle-7.3.zip
+        unzip /opt/gradle/gradle-${{ matrix.gradle-version }}.zip -d /opt/gradle
```

Comment:
Thoughts on using https://github.com/gradle/gradle-build-action to clean this up a bit?

Reply:
Added gradle-build-action to the CI. Confirmed it's working on the GH runner!